hoomd/md/pair/pair.py

```diff
@@ -1659,7 +1653,7 @@ class LJ1208(Pair):
         \\begin{eqnarray*}
         V_{\\mathrm{LJ}}(r)
           = & 4 \\varepsilon \\left[ \\left( \\frac{\\sigma}{r} \\right)^{12} -
-          \\alpha \\left( \\frac{\\sigma}{r} \\right)^{8} \\right]
+          \\left( \\frac{\\sigma}{r} \\right)^{8} \\right]
```

Comment:
The changes to documentation of potential parameters seem unrelated to the purpose of this PR. This PR is very large and I would advocate for smaller PRs in the future, because I can't review this in one pass.

Reply:
Yes this one is unrelated. I ended up fixing it in this PR because I only noticed the error due to other changes I was making.